packages/zowe-explorer/README.md

```diff
@@ -358,6 +358,8 @@ For more information about SCS, see [Secure Credential Store Plug-in for Zowe Ex
 
 - **Associate profiles**: You can create a secondary association by right-clicking the profile and selecting the **Associate profiles** icon. For more information, see [the Associate profiles section](https://docs.zowe.org/stable/user-guide/ze-profiles.html#associate-profile) in Zowe Docs.
 
+  > **Note**: The Associate Profile functionality is deprecated and will be removed in the version 2 of Zowe Explorer. Zowe Explorer V2 is slated for February 2022.
+
```

Comment:
@IgorCATech Just re-direct them [here](https://www.zowe.org/download.html) for the release timeline

Reply:
As @lauren-li suggested, and I completely support the suggestion, it would be good to add the anchor link that lands on the appropriate heading on that page. I'll need to add an ID to that heading to be able to accomplish this task. We can either wait for me to do that, then merge this PR, or merge this PR first and, once the HTML page on the Zowe website is fixed, create a new PR with the small fix. 
If this Deprecated note can wait, I will go for the former option. Let me know what works better for you. Thanks

Reply:
@IgorCATech Unless the Zowe website will be updated soon, I would go ahead and merge in this PR and then create another PR when the fix is in. It is more important that we give any users of Associated Profiles a timely deprecation warning, and plus we already describe in this PR update what section of the Download page to look for.

The anchor link to the release timeline section will be a nice enhancement to increase convenience for our users when it is ready. Thanks for looking into it!

Reply:
https://github.com/zowe/zowe.github.io/pull/271 - PR opened